apps/L1G/restaurant/restaurant-frontend/src/utils/ImageDownloader.ts

```
@@ -0,0 +1,6 @@
+export const ImageDownloader = (base64Data: string, filename = 'image.png') => {
```

ESLint

[error] Filename is not in kebab case. Rename it to `image-downloader.ts`.

(unicorn/filename-case)